deploy/scripts/init_certs.sh

```diff
@@ -13,6 +13,60 @@ else
     exit 0
 fi
 
-# Generate certificates
-BASE_DOMAIN=$(python3 "$YAML_HELPER" extract .global.baseDomain "$CONFIG_FILE")
-/usr/share/flightctl/generate-certificates.sh --cert-dir "$CERT_DIR" --api-san "$BASE_DOMAIN"
+# Get the system hostname FQDN for use as default and in certificate SANs
+hostname_fqdn=$(hostname -f || hostname)
+
+# Get the primary host IP address:
+# First try to get the IP address used to reach the internet.
+# If that fails (e.g. no internet access), use the first non-loopback IP.
+host_ip="$(
```

Comment:
why should we only extract the first / internet facing one ?
wouldnt just takeing hostname -I be a better choice ( this would make it work in more complicated scnearios where there are multiple ips etc)

Reply:
It's a slippery slope. That list would also contain, for example, the Podman bridge IP, so we'd need to filter that out reliably. It may also leak more informations about the internal networks than comfortable and include transient IP addresses...

Reply:
I agree this is a hard decision but it was the choice to make the system deterministic. first interface first ip.

Reply:
Ok, I've implemented a compromise where the default (routable) interface is being used if it exists and if it doesn't, we fall back to `hostname -I` and add all returned interfaces (as we cannot assume the first to be the correct one). Either way, users are free to create their own certificates if needed.